signal_scanner_bot/env.py

```diff
@@ -42,6 +42,13 @@ def log_vars() -> None:
 TWITTER_ACCESS_TOKEN = _env("TWITTER_ACCESS_TOKEN")
 TWITTER_TOKEN_SECRET = _env("TWITTER_TOKEN_SECRET")
 TRUSTED_TWEETERS = set(str(_env("TRUSTED_TWEETERS", default="")).split(","))
+SEND_HASHTAGS = str(_env("SEND_HASHTAGS", default="")).split(",")
+RECEIVE_HASHTAGS = str(_env("RECEIVE_HASHTAGS", default="")).split(",")
+SIGNAL_MESSAGE_HEADERS = set(str(_env("SIGNAL_MESSAGE_HEADERS", default="")).split(","))
+
+# Check for testing env var and update other vars as necessary
+TESTING = True if str(_env("TESTING", default="false")).lower() == "true" else False
+DEBUG = True if TESTING else DEBUG
```

Comment:
Defining this twice is a little interesting haha, perhaps we can have this be
```
DEBUG = TESTING or _env(<debug stuff>)
```
and only define it once?

Reply:
Yeah I did this dumbly, updated to define TESTING first then have DEBUG set based on the value of TESTING.